crates/rpc/src/lib.rs

```diff
@@ -129,7 +129,11 @@ impl Handler {
 
     async fn accounts(_: Params, _: Ctx) -> jsonrpc_core::Result<serde_json::Value> {
         let wallets = Wallets::read().await;
-        let address = wallets.get_current_wallet().get_current_address().await;
+        let address = wallets
+            .get_current_wallet()
+            .ok_or(jsonrpc_core::Error::internal_error())?
```

Comment:
the error coming from here should include more info. not just an "internal error". ideally you should add something to `iron_rpc::Error`

Reply:
Agree, but given the return type this was the only error that I found suitable